R/est.incidence.by.R

```diff
@@ -101,6 +101,27 @@ est.incidence.by <- function(
     return(to_return)
   }
 
+  if (!any(is.element(strata, pop_data %>% names()))) {
+    # search strata variable from pop_data
+    strata_var <-
+      grep(
+        x = pop_data %>% names(),
+        value = TRUE,
+        pattern = strata,
+        ignore.case = TRUE
+      )
+
+    cli::cli_abort(
+      class = "missing_var",
+      message = c(
+        "Can't find the column {.arg {strata}} in {.arg pop_data}.",
```

Comment:
Please inform the user that this error relates to their input for the `strata` argument, so they know which input to fix.

Reply:
This has been implemented.